cmd/state-svc/main.go

```diff
@@ -231,7 +231,7 @@ func runStatus(out output.Outputer) error {
 
 	out.Print(fmt.Sprintf("Port: %s", port))
 	out.Print(fmt.Sprintf("Dashboard: http://127.0.0.1%s", port))
-	out.Print(fmt.Sprintf("Log: %s\n", logging.FilePathFor(logging.FileNameFor(os.Getpid()))))
+	out.Print(fmt.Sprintf("Log: %s\n", cfg.GetString(constants.SvcLogConfig)))
```

Comment:
I feel fairly strongly that we should not be using the config db for IPC, and that this info should be passed through the authoritative instance of the service. In other words, I still think the service will likely benefit from having a PID tcp-endpoint, or we should just not report this info. @Naatan did indicate we can drop it and wait to see if we reach more of a need for that endpoint, but pinging him to be sure.

Reply:
Actually I just ran into a use-case for this myself: integration tests.

How trivial would it be to employ IPC for this? I agree it sounds like a better solution, just wondering about the cost vs value picture.

Reply:
Quite trivial. I can make a story with the AC being clear steps to take and it should take very little time.

Reply:
Ok, then let's go the IPC route.

Reply:
Just update the current story though, don't need a new one.